include/SFML/Graphics/View.hpp

```diff
@@ -131,6 +131,19 @@ class SFML_GRAPHICS_API View
     ////////////////////////////////////////////////////////////
     void reset(const FloatRect& rectangle);
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Reset the view to the given center and size
+    ///
+    /// Note that this function resets the rotation angle to 0.
+    ///
+    /// \param center Center of the zone to display
+    /// \param size   Size of zone to display
+    ///
+    /// \see setCenter, setSize, setRotation
+    ///
+    ////////////////////////////////////////////////////////////
+    void reset(const Vector2f& center, const Vector2f& size);
```

Comment:
I think this addition is out of the scope of this PR.

Reply:
Removed